assets/src/dashboard/components/dropdown/index.js

```diff
@@ -39,24 +41,29 @@ const Label = styled.label`
 
 export const InnerDropdown = styled.button`
   align-items: center;
-  background-color: ${({ theme, transparent }) =>
-    transparent ? 'transparent' : theme.colors.gray25};
-  border-radius: 4px;
-  border: 1px solid transparent;
+  background-color: ${({ theme, type, isOpen }) =>
+    theme.dropdown[type][isOpen ? 'activeBackground' : 'background']};
+  border-radius: ${({ theme, type }) => theme.dropdown[type].borderRadius};
+  border: ${({ theme, type }) => theme.dropdown[type].border};
```

Comment:
Since there are 3 different styles, it seemed better to use a theme rather than duplicate 90% of the same logic and CSS code to create 3 variants. Most of the differences are around color and border.

Reply:
I like how readable this is! 

nit: proptypes for styledComponents using props other than theme.